<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* background: url(/home/matt/Documents/imgpy/mantidimaging/gui/ui/code.jpg)*/</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReconstruct">
    <property name="title">
     <string>Reconstruct</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCenter_of_Rotation"/>
    <addaction name="actionManual_Center_of_Rotation"/>
    <addaction name="separator"/>
    <addaction name="actionTomopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuReconstruct"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTomopy">
   <property name="text">
    <string>Tomopy</string>
   </property>
  </action>
  <action name="actionCenter_of_Rotation">
   <property name="text">
    <string>Center of Rotation</string>
   </property>
  </action>
  <action name="actionManual_Center_of_Rotation">
   <property name="text">
    <string>Manual Center of Rotation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
